### ISSUE TITLE
Error: Error: Command failed: git push heroku HEAD:refs/heads/main --force. 'heroku' does not appear to be a git repository

### ISSUE THREAD

**Issue by mattfrancis888** [AkhileshNS/heroku-deploy#107]
Error I am getting in Github actions:

![image](https://user-images.githubusercontent.com/22691408/141732238-13182859-9e7a-4a95-bad0-a063a044dc75.png)

I think it might be an unfixed bug based on this issue:
https://github.com/AkhileshNS/heroku-deploy/issues/84

However, in the case that it is not, here is 

Steps on what I did:
1. I made sure that HEROKU_API_KEY is a secret in the github repo:
![image](https://user-images.githubusercontent.com/22691408/141732661-bddb7d89-1483-4d03-899d-208761267d41.png)

2. Pushed local repo to heroku remote repo
`heroku git:remote -a gh-actions-heroku
git push heroku main`

Proof that:
gh-actions-heroku is the app in my Heroku dashboard
![image](https://user-images.githubusercontent.com/22691408/141732897-0840ef5f-ede5-4a79-a353-76bf7ff2ec3f.png)

Proof that it was pushed: 
![image](https://user-images.githubusercontent.com/22691408/141732980-9d598747-e9c9-4842-8915-6244552bbc0e.png)

Yet I am still receiving the issue. Am I doing anything wrong?

**Comment by fabianobonomini** [comment]
same problem here

**Comment by dzcpy** [comment]
Aame issue. What's the solution？

**Comment by danielricecodes** [comment]
I just ran into this problem; however my HEROKU_API_KEY had been set and working for nearly 6 months.  I suspect Heroku may have auto-rotated my API key because all I did was update the key and now this script is working again. 

TL;DR: Visit https://dashboard.heroku.com/account and update your key on Github.
